spin/cmds/meson.py

```diff
@@ -124,6 +134,31 @@ def _meson_version_configured():
         pass
 
 
+def _check_coverage_tool_installation(coverage_type: GcovReports):
+    requirements = {  # https://github.com/mesonbuild/meson/blob/6e381714c7cb15877e2bcaa304b93c212252ada3/docs/markdown/Unit-tests.md?plain=1#L49-L62
+        GcovReports.html: ["Gcovr/GenHTML", "lcov"],
```

Comment:
Good to know this works :) https://github.com/scientific-python/spin/actions/runs/7677258874/job/20925774260?pr=159#step:5:212